events/events-team/sig-contrib-events.md

```diff
@@ -0,0 +1,71 @@
+# SIG Contributor Events
+
+This document defines planning and activities needed to organize the SIG Meet and Greet and SIG F2F activities at Contributor Summits.
+
+- [SIG Meet and Greet](#sig-meet-and-greet)
+- [SIG F2Fs](#sig-f2fs)
+
+## Skills and Qualifications
+
+- Familiarity with the SIGs, including understanding how SIGs are related to each other and any likely overlap in membership.
+- Good organizational skills, including being able to relentlessly follow up and get responses if SIG leads are unresponsive.
+
+## SIG Meet and Greet
+
+### Overview
+
+The SIG Meet and Greet is a great place for SIGs to recruit contributors. As part of the Contributor Summits, SIGs will have easy access to experienced
+contributors interested in getting involved in new SIGs along with brand new contributors who are excited to get started in Kubernetes 
```

Comment:
I think worth pointing out that not all new contributors are recent college grads struggling with Go; maybe present the potential options as a list?